docs/components/content/GoogleMapsDemo.vue

```diff
@@ -1,17 +1,39 @@
 <script setup lang="ts">
 import { ref } from 'vue'
-import type { Ref } from 'vue'
 
 const isLoaded = ref(false)
 const center = ref()
 const maps = ref()
 
-const query = ref('Space+Needle,Seattle+WA')
-function handleReady(_map: Ref<google.maps.Map>) {
-  const map = _map.value
-  center.value = map.getCenter()
-  map.addListener('center_changed', () => {
-    center.value = map.getCenter()
+const query = ref({
+  lat: -37.7995487,
+  lng: 144.9867841,
+})
+
+const markers = ref([])
+
+let increment = 1
+function addMarker() {
+  // push to markers, we want to add a marker from the center but randomize the position by a bit
```

Comment:
what's the reasoning for randomizing? It's not clear without context.

Reply:
This is just a demo, it's to showcase the reactivity